README.md

```diff
@@ -23,47 +24,17 @@ Liqo leverages the same highly successful “peering” model of the Internet, w
 * Seamless pod execution on remote cluster,
 * Seamless reconciliation on remote clusters of K8s objects (i.e. configmaps, secrets, services, endpoints)
 
-
 ## Quickstart
 
-Liqo can be installed via Helm. 
-
-### Pre-requisites
-
-* Two Kubernetes clusters
-    * Supported CNIs:
-      * Flannel
-      * Calico 
-    * K3s is also supported
-* Helm 3
-
-### Installation
-
-The following process will install Liqo on your Cluster. This will make your cluster ready to share resources with other Liqo resources.
-
-#### Kubernetes
-
-Liqo Installer should be capable to look for the cluster parameters required. 
+Liqo can be installed via Helm. Liqo Installer should be capable to look for the cluster parameters required on cluster
```

Comment:
The parameters of the home cluster required by Liqo to start should be automatically discovered by the Liqo installer (through proper `kubeadm` calls).

Reply:
Additional comment. Why are we saying that Liqo can be installed via "helm", then we list a command that is a bash script?